core/admin.py

```diff
@@ -1,11 +1,41 @@
 from django.contrib import admin
 from django_admin_display import admin_display
 from django.utils.safestring import mark_safe
+import os
 
 from . import models
 from . import tasks
 
 
+# should take the FileField object rather than the model.
+# a module for log preview
+def _text_preview(log_file):
```

Comment:
It is more common to document functions using a doc string.  That is, rather than a comment above the function, this would be:
```
def _text_preview(log_file):
    """
    Return the text of a file if it is short or the last portion of it if it is long.

    :params: log_file A FileField to read text from.
    """
```

Alternately, since we are using a new enough version of Python, you can use type decorators (e.g.,
```
def _text_preview(log_file: FileField):
```

Reply:
I got a "NameError: name 'FileField' is not defined" so I changed some code to work around it and lint test. But I am not sure if I made the correct fix.